app/javascript/components/publicHealth/Import.js

```diff
@@ -120,52 +126,48 @@ class Import extends React.Component {
           return (
-            <div className="alert alert-dismissible fade show alert-danger mt-3 mx-3" role="alert" key={`error${index}`}>
+            <Alert key={index} variant="danger" className="mt-3 mx-3">
               {error}
-              <button type="button" className="close" data-dismiss="alert" aria-label="Close">
-                <span aria-hidden="true">&times;</span>
-              </button>
-            </div>
+            </Alert>
           );
         })}
         {this.state.errors.length == 0 && (
-          <div className="m-4">
-            {this.state.phased.length > 0 && (
-              <div className="progress mb-3">
-                <div
-                  className="progress-bar progress-bar-striped progress-bar-animated"
-                  role="progressbar"
-                  aria-valuenow={this.state.progress}
-                  style={{ width: Math.round((this.state.progress + 1 / this.state.phased.length) * 100) + '%' }}
-                  aria-valuemin="0"
-                  aria-valuemax={this.state.phased.length}></div>
-              </div>
-            )}
+          <div className="mx-3 mt-1 mb-2">
             <h5>
               Please review the monitoree records that are about to be imported. You can individually accept or reject each record below. You can also choose to
               import all unique records or all records (including duplicates) by clicking the &quot;Import All&quot; button.
             </h5>
-            <Button
-              variant="primary"
-              className="btn-lg my-2"
-              onClick={() =>
-                this.handleConfirm(
-                  `This will import all records listed below that you did not manually accept or reject. If potential duplicates have been detected, check the box if you would like to import them.`
-                )
-              }>
-              Import All
-            </Button>
+            {this.state.acceptedAllStarted ? (
+              <Button variant="primary" className="btn-lg mt-2" disabled="true">
+                <i className="fas fa-upload"></i> Import All
+              </Button>
+            ) : (
+              <Button
+                variant="primary"
+                className="btn-lg mt-2"
+                onClick={() =>
+                  this.handleConfirm(
+                    `This will import all records listed below that you did not manually accept or reject. If potential duplicates have been detected, check the box if you would like to import them.`
+                  )
+                }>
+                <i className="fas fa-upload"></i> Import All
+              </Button>
+            )}
+
             {this.state.acceptedAllStarted && (
-              <Button variant="primary" className="btn-lg my-2 ml-2" onClick={() => this.stopImport()}>
-                Stop Import
+              <Button variant="danger" className="btn-lg mt-2 ml-2" onClick={() => this.stopImport('Stop Import')}>
```

Comment:
If there's a chance we may re-word `Stop Import` later down the road, why not define all instances of it as a global constant? `const STOP_IMPORT_BUTTON_TEXT = "Stop Import"`
If not, then why not just hardcode `Stop Import` into the `stopImport()` function instead of passing it around as a String ?

Reply:
Ahh I didnt see it had been included elsewhere